express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+const grades = [];
+
+const getJson = app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+const jsonParse = express.json(getJson);
+app.use(jsonParse, (req, res) => {
+  const idNum = grades.length + 1;
```

Comment:
There should be another variable near grades named `nextId`. That will be your id

Reply:
Express Post JSON Id Update | Yang An Boada
e3214e5